.golangci.yml

```diff
@@ -6,21 +6,50 @@ run:
 
 linters:
   enable:
-    - protogetter # reports direct reads from proto message fields when getters should be used.
-  # Right now revive raises 50+ issues, whereas golint didn't have any.
-  # despite revive being recommended as a replacement for golint.
-  # TODO: should we turn on revive and fix the issues?
-  #     - revive # drop-in replacement for golint
-  exclusions:
+    - protogetter # reports direct reads from proto message fields when getters should be used
+    - revive      # drop-in replacement for golint
+    - lll         # enforces line length limits
+    - errorlint   # makes sure errors are wrapped correctly
+    - misspell    # checks for common misspellings
```

Comment:
oh wut :O can't wait to see if this is good or not

Reply:
I would definitely be against it if it was a "normal" spellcheck. But it seems written to just check for common, unambiguous misspellings. I've high hopes for this one